public/js/services/keyboard_navigation_service.js

```diff
@@ -66,75 +66,83 @@ export function keyNav (event, pulsate, preserveIndex) {
   }
 }
 
-Mousetrap.bind(['left', '['], function () {
-  clearTargets()
-  currentIndex--
-  if (currentIndex < 0) {
-    currentIndex = targetsLength - 1
-  }
-  $(targets[currentIndex]).addClass('keynav-target')
-})
+Mousetrap.pause = function () {
+  Mousetrap.reset()
+}
 
-Mousetrap.bind(['right', ']'], function () {
-  clearTargets()
-  currentIndex++
-  if (currentIndex >= targetsLength) {
-    currentIndex = 0
-  }
-  $(targets[currentIndex]).addClass('keynav-target')
-})
+Mousetrap.unpause = function () {
```

Comment:
Appreciate the fix, but why add `unpause` and `pause` to the Mousetrap object itself?
It's best to avoid adding methods to a library that preform operations specific to a specific application.
It seems more appropriate to scope them directly to the `keyNav` module.

But, it looks like this extension has another approach that would let us avoid rebinding everything.
https://github.com/ccampbell/mousetrap/blob/master/plugins/pause/mousetrap-pause.js
Could that work?

Reply:
@buck54321 ^ we can go back to your original PR to iron out issues.  I just wanted to get a PR ready in case it was good to go.  Yours just needs a rebase I think.

Reply:
Sounds good. Mousetrap pause adds the methods directly to Mousetrap too, but with a little more tact. https://github.com/ccampbell/mousetrap/blob/master/plugins/pause/mousetrap-pause.js